cranelift/codegen/src/ir/function.rs

```diff
@@ -295,20 +295,12 @@ impl FunctionStencil {
                 }
             }
 
-            InstructionData::BranchTable {
-                table,
-                destination: default_dest,
-                ..
-            } => {
+            InstructionData::BranchTable { table, .. } => {
                 self.dfg.jump_tables[*table].iter_mut().for_each(|entry| {
                     if *entry == old_dest {
                         *entry = new_dest;
                     }
                 });
-
-                if *default_dest == old_dest {
-                    *default_dest = new_dest;
-                }
```

Comment:
This is still needed, right?

Reply:
As it's using `iter_mut`, the default label is included.

Reply:
I see

Reply:
I switched it to iterate over `all_branches_mut` instead, which reads a bit better and doesn't change the meaning of the old function.